src/zone.js

```diff
@@ -429,6 +430,102 @@ Zone.prototype.createDisk = function(name, config, callback) {
   );
 };
 
+/**
+ * Create an instance group manager in this zone.
+ *
+ * @see [InstanceGroupManager Resource]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers#resource}
+ * @see [InstanceGroupManagers: insert API Documentation]{@link https://cloud.google.com/compute/docs/reference/v1/instanceGroupManagers/insert}
+ *
+ * @param {string} name - Name of the instance group manger.
+ * @param {config=} config - Configuration object.
+ * @param {instanceTemplate} config.instanceTemplate - Instance template to use for this instance group manager.
```

Comment:
We have to capitalize to `@param {InstanceTemplate}` so the docs can create a link to the type.